jobs/build/reposync/Jenkinsfile

```diff
@@ -74,7 +66,11 @@ node {
 
             stage("push to mirror") {
                 sh "rsync -avzh --delete -e \"ssh -o StrictHostKeyChecking=no\" ${LOCAL_SYNC_DIR} ${MIRROR_TARGET}:${MIRROR_PATH} "
-                buildlib.invoke_on_use_mirror("push.enterprise.sh")
+                mirror_result = buildlib.invoke_on_use_mirror("push.enterprise.sh")
+                if (mirror_result.contains("[FAILURE]")) {
+                    echo mirror_result
+                    throw "Error running ${SYNC_VERSION} reposync push.enterprise.sh, see details above."
```

Comment:
I think `throw` has to take an exception... can this be `error()` instead?

Reply:
Oh, and could the result go in the error? That way it would go in the email too and you don't have to click to the console to see what happened...

Reply:
Fixed.